agenta-backend/agenta_backend/tasks/evaluations.py

```diff
@@ -78,11 +78,20 @@ def evaluate(
                     fetch_evaluator_config(evaluator_config_id)
                 )
 
+                additional_kwargs = (
+                    {
+                        "app_params": app_variant_db.config.parameters,
+                        "inputs": data_point,
```

Comment:
we better fetch these from the config like the line above (app_variant_db.config.parameters.inputs) not from the testset.
but as there is a [bug](https://github.com/Agenta-AI/agenta/issues/1102) right now with the inputs we might not able to do it.
so let's simply add a comment that we need to fetch it from app_variant_db.config.parameters.inputs once the bug is fixed